docs/generators/java.md

```diff
@@ -368,3 +368,10 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |XML|✓|OAS2,OAS3
 |PROTOBUF|✗|ToolingExtension
 |Custom|✗|OAS2,OAS3
+
+## 🔧 Example: Using Maven CLI to view generator version
+
+To check which version of the OpenAPI Generator Maven plugin is configured in your project, use:
+
+```bash
+mvn openapi-generator:version
```

Comment:
this doc is for the java client generator

for maven plugin doc, please refer to https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator-maven-plugin/README.md

feel free to file a PR to update this readme to improve the maven plugin doc